micro_ros_setup/config/freertos/olimex-stm32-e407/create.sh

```diff
@@ -0,0 +1,18 @@
+pushd $FW_TARGETDIR >/dev/null
+    # Install toolchain
+    mkdir toolchain
+    
+    echo "Downloading ARM compiler, this may take a while"
+    curl -fsSLO https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2019q3/RC1.1/gcc-arm-none-eabi-8-2019-q3-update-linux.tar.bz2 
+    tar --strip-components=1 -xvjf gcc-arm-none-eabi-8-2019-q3-update-linux.tar.bz2 -C toolchain  > /dev/null
```

Comment:
On NuttX, we're using the default Debian package for the toolchain. If this is not good, maybe we should change it in general, not just for FreeRTOS.

Reply:
In FreeRTOS + Olimex STM32 E407 is not a problem to use the default Debian GCC ARM toolchain. But when building Crazyflie version the Crazyflie firmware uses a new version:

https://github.com/bitcraze/crazyflie-firmware#debianubuntu

Maybe we can left the Crazyfie version with this newer toolchain and use the Debian version by default when possible. What do you think?

Reply:
Yeah, that sounds good for now.

That said, I've noticed that the ARM version of their toolchain is *much* newer. With our current version, we have the atomics issue -- hence the libatomics.c **hack**. It would be interesting to know whether using a newer toolchain would solve that issue.